canonical-web-and-design/jaas-dashboard
# Add machine table to model details page

## Done

Added a table for the machine data in the models details page.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- View a model with machines, you should see their data.

## Details

Fixes #126 

## Known issues

Sometimes the message content can be very large, an issue has been raised with design.
